src/libraries/System.Data.Common/src/System/Data/Common/DbDataAdapter.cs

```diff
@@ -231,6 +232,7 @@ protected virtual void ClearBatch()
             throw ADP.NotSupported();
         }
 
+        [UnsupportedOSPlatform("browser")]
```

Comment:
What makes this unsupported on browser?

Reply:
Right, it looks like a false-positive. Thanks!